collections/Laud_Misc/MS_Laud_Misc_132.xml

```diff
@@ -37,34 +37,76 @@
                      <idno type="SCN">1565</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>Augustine, <title>Enarrationes in Psalmos 119-133</title>; Germany (Lorsch), 9th century, first half</head>
```

Comment:
Do we publish somewhere what needs to go into the `<head>` element?

Reply:
https://msdesc.github.io/consolidated-tei-schema/msdesc.html#heading